server/src/main/java/io/crate/expression/scalar/systeminformation/ColDescriptionFunction.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.systeminformation;
+
+import io.crate.data.Input;
+import io.crate.expression.scalar.ScalarFunctionModule;
+import io.crate.metadata.FunctionName;
+import io.crate.metadata.NodeContext;
+import io.crate.metadata.Scalar;
+import io.crate.metadata.TransactionContext;
+import io.crate.metadata.functions.BoundSignature;
+import io.crate.metadata.functions.Signature;
+import io.crate.metadata.pgcatalog.PgCatalogSchemaInfo;
+import io.crate.types.DataTypes;
+
+public final class ColDescriptionFunction extends Scalar<String, Object> {
+
+    private static final String NAME = "col_description";
+    private static final FunctionName FQN = new FunctionName(PgCatalogSchemaInfo.NAME, NAME);
+
+    public static void register(ScalarFunctionModule module) {
+        module.register(
+            Signature.scalar(
+                FQN,
+                DataTypes.INTEGER.getTypeSignature(),
+                DataTypes.INTEGER.getTypeSignature(),
+                DataTypes.STRING.getTypeSignature()
+            ),
+            ColDescriptionFunction::new
+        );
+    }
+
+    private final Signature signature;
+    private final BoundSignature boundSignature;
+
+    public ColDescriptionFunction(Signature signature, BoundSignature boundSignature) {
+        this.signature = signature;
+        this.boundSignature = boundSignature;
+    }
+
+    @SafeVarargs
+    @Override
+    public final String evaluate(TransactionContext txnCtx, NodeContext nodeCtx, Input<Object>... args) {
+        // CrateDB doesn't support comments for table columns, so always return null
+        return null;
+    }
+
+    @Override
+    public Signature signature() {
+        return signature;
+    }
+
+    @Override
+    public BoundSignature boundSignature() {
+        return boundSignature;
+    }
+}
```

Comment:
We can use `BinaryScalar`, smith like

```
 module.register(
                Signature.scalar(
                    FQN,
                    DataTypes.INTEGER.getTypeSignature(),
                  DataTypes.INTEGER.getTypeSignature(),
                  DataTypes.STRING.getTypeSignature()
                ).withFeatures(NO_FEATURES),
                (signature, boundSignature) ->
                    new BinaryScalar<>((a, b) -> null, signature, boundSignature, DataTypes.LONG) 
```

Reply:
Pretty sure this would need a refactor of the BinaryScalar as it requires all inputs and outputs to be of the same type

Reply:
true:
```
All bound argument types of the signature must match the type argument
java.lang.AssertionError: All bound argument types of the signature must match the type argument
	at __randomizedtesting.SeedInfo.seed([7ECA45A9BD1B8F49:ED248C8B447329BC]:0)
	at io.crate.expression.scalar.arithmetic.BinaryScalar.<init>(BinaryScalar.java:45)
	at io.crate.expression.scalar.systeminformation.ObjDescriptionFunction.lambda$register$1(ObjDescriptionFunction.java:50)
```

Reply:
> Pretty sure this would need a refactor of the BinaryScalar implementation

Sorry, I missed that.